### DISCUSSION TITLE
Universe 2022:  What are some uses of Codespaces you can think of for education?

### DISCUSSION THREAD

**Discussion by j0ashm** [community/community#38783] [Codespaces]
### Select Topic Area

General

### Body

At Universe, we heard about how Codespaces can empower teachers and students by lowering the barrier of entry.
Building from that, I’m curious what the community thinks about Codespaces, and what applications you see for it in in your classrooms?

**Comment by ossanna16** [comment]
Hi there @j0ashm and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
